README.md

```diff
@@ -47,3 +47,13 @@ Run a simple 1-bit adder design using the [XLS](https://google.github.io/xls/) h
 - Target the open source [SKY130](https://github.com/google/skywater-pdk/) PDK to manufacture your design using Google's sponsored [OpenMPW shuttle](https://developers.google.com/silicon) program.
 
 ![xls layout](img/xls.png)
+
+### Different adder architecture with OpenLane [![colab](https://colab.research.google.com/assets/colab-badge.svg)](https://colab.research.google.com/github/heyshakya/silicon-notebooks/blob/main/adder_implementation.ipynb)
```

Comment:
s/heyshakya/chipsalliance/

Reply:
This PR is also removing `xls-workshop-openlane-ja.ipynb` is that intentional?
Please ignore that change.
I have no intention to delete any existing files from the repo.
I just want to add a few.
I did that for my own repo.